### PR TITLE
Helm add-on

### LINKED ISSUE #1: Create Helm addon
Create an add-on that enables the following:

- [x] RBAC roles and rolebindings for helm and tiller so that helm can install applications via `helm install ....` commands by default.
- [x] helm and tiller [enabled](https://docs.helm.sh/using_helm/#quickstart) for an EKS cluster

Both tasks should be accomplished by running a `Makefile` after the EKS cluster has been created, similar to the k8s dashboard in `/dashboard`.

### PR DESCRIPTION
**Pull request by tlake**
# About
Once a Kubernetes cluster has been configured and is set up to be the current cluster, this add-on installs the `helm` client to your local machine and `tiller` in the Kubernetes cluster. Note that in the current implementation `tiller` is installed to a service account (named "tiller") which is bound to the `cluster-admin` cluster role.

Installation is achieved by running `make install` from the `helm/` directory.

# Issues

Closes #1

### CODE REVIEW

File: helm/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+create-service-account:
+	kubectl apply -f tiller-cluster-role-binding.yaml
+
+install: create-role-binding create-service-account install-helm install-tiller
+
+install-helm:
+	curl -s https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+	chmod 700 get_helm.sh
+	@./get_helm.sh
```
Comment (diemonster) [inline helm/Makefile]: [nit] consider adding a `clean` target to remove this script after it's run?

Comment (tlake) [reply helm/Makefile]: ✅

File: helm/Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+create-role-binding:
```
[the remark this answers is not in the document]
Comment (tlake) [reply helm/Makefile]: ✅

File: helm/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (>= v1.10.4) installed, found in your local machine's PATH, and configured to the desired cluster.
+
+## Walkthrough
+Run the following command(s) from this directory:
+
+1. Use the Makefile to install Helm.
+**You'll be prompted to input your sudo password.**
+```console
+$ make install
+kubectl apply -f tiller-service-account.yaml
```
[the remark this answers is not in the document]
Comment (tlake) [reply helm/README.md]: Nice, yeah - I also don't have a defined sense of when long output is too much vs when it's good to give the user a sense of "yeah, I did this right." I'm happy to elide here if that makes sense to you.

File: helm/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Helm Add-on
+This add-on will enable Helm for your Kubernetes cluster.
+The [`helm`](https://docs.helm.sh/glossary/#helm-and-helm) client will be installed locally, and the [`tiller`](https://docs.helm.sh/glossary/#tiller) server will be installed into the cluster.
+
+**NOTE: Currently, this implementation installs Helm and tiller into a cluster-wide `tiller` service account.
+More work will need to be done in the future to properly configure more stringent RBAC setitngs.**
```
[the remark this answers is not in the document]
Comment (tlake) [reply helm/README.md]: ✅

File: helm/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tlake) [reply helm/README.md]: ✅

File: helm/README.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+$HELM_HOME has been configured at <<$HOME>>/.helm.
+
+Tiller (the Helm server-side component) has been installed into your Kubernetes Cluster.
+
+Please note: by default, Tiller is deployed with an insecure 'allow unauthenticated users' policy.
+For more information on securing your installation see: https://docs.helm.sh/using_helm/#securing-your-helm-installation
+Happy Helming!
+```
+
+2. Install things!
```
[the remark this answers is not in the document]
Comment (tlake) [reply helm/README.md]: Oooh, good call - to my knowledge, we don't have any other helm-related add-ons in this repo _yet,_ but I'll definitely add a link to the hub.

✅

File: helm/Makefile (hunk 1 above)
Comment (tlake) [reply helm/Makefile]: Hm - I can see an argument for both `clean` and `uninstall` targets to exists together, where `clean` gets rid of interstitial files used only during installation, and `uninstall` actually does the whole  teardown and `terraform destroy`. Thoughts?

Comment (zpatrick) [reply helm/Makefile]: I think `uninstall` definitely needs to exist. 

I'm a little wary about `clean` b/c it adds more work for us and for future addons if we make that a requirement, its adds size/complexity to the Makefile (which I think should be as simple and minimalist as possible), and doesn't have a very clear scope (do we remove the `.terraform` dir that gets built during init? the helm binary we installed? etc.). 

At least with `uninstall` the scope is clear: run `terraform destroy`, `helm uninstall`, `kubectrl uninstall` (or whatever the command is) against anything done in the `install` target. 

All that being said, my reasons more come from personal preference than towards objective "wins", so consider my comments regarding this to be [nit]

**Pull request merged**